recipes-soletta/soletta/soletta_git.bb

```diff
@@ -21,11 +22,15 @@ S = "${WORKDIR}/git"
 
 inherit cml1 python3native
 
+PACKAGECONFIG ??= "${@bb.utils.contains('DISTRO_FEATURES','x11', 'x11', '', d)}"
+PACKAGECONFIG[x11] = "cairo atk gtk+3 gdk-pixbuf pango"
```

Comment:
Alright I will put in the same format as you put in the patch: https://github.com/solettaproject/meta-soletta/pull/79 and run some more tests